docs/content/faq.md

```diff
@@ -82,9 +82,8 @@ of metadata, which breaks the desired 1:1 mapping of files to objects.
 
 ### Can rclone do bi-directional sync? ###
 
-No, not at present.  rclone only does uni-directional sync from A ->
-B. It may do in the future though since it has all the primitives - it
-just requires writing the algorithm to do it.
+Yes, since rclone v1.58.0, experimental bidirectional cloud sync is
```

Comment:
Well spotted!

Can you add a link to it like this too please? And we'll leave the experimental bit to the following page

     Yes, since rclone v1.58.0, [bidirectional cloud sync](/bisync/) is

Reply:
Hi, I've modified my 'patch-1' branch based on your suggestion 😃